test/test.js

```diff
@@ -0,0 +1,38 @@
+const assert = require('assert');
+const app = require('../index');
+const axios = require('axios');
+
+const localUrl = 'http://localhost:8000/';
+describe('testing', function () {
```

Comment:
since the port is configurable you can use `config.port` instead of hard-coding 8000